techlibs/xilinx/brams_init.py

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env python3
 
-with open("techlibs/xilinx/brams_init_18.vh", "w") as f:
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
```

Comment:
This is a python 3 script. Those `from __future__ import` lines are unnecessary.